common/app/views/fragments/newHeader.scala.html

```diff
@@ -11,7 +11,7 @@
         ) { case (currentTopLevelSection) =>
             <div class="new-header__inner gs-container">
                 <a href="@LinkTo{/}" class="new-header__logo-wrapper" tabindex="0" data-link-name="nav2 : logo">
-                    <h1 class="u-h">The Guardian</h1>
+                    <span class="u-h">The Guardian</span>
```

Comment:
One of the first things I did, when I started was to change the text of the logo into 'The Guardian - home', so maybe we want to do this here aswell for a11y reasons.